ocaml/parsing/pprintast.ml

```diff
@@ -1766,14 +1766,14 @@ and type_def_list ctxt f (rf, exported, l) =
                  (list ~sep:"@," (type_decl "and" Recursive)) xs
 
 and record_declaration ctxt f lbls =
-  let pld_mode_type_attrs pld =
+  let pld_mode_type_attrs pld : _ format6 * _ =
     match Jane_syntax.Constructor_argument.of_ast pld.pld_type with
-    | Some (Jcarg_local (Lcarg_global carg)) -> "global_ ", carg
+    | Some (Jcarg_local (Lcarg_global carg)) -> "global_@;", carg
     | None -> "", pld.pld_type
   in
   let type_record_field f pld =
     let mode_flag, pld_type = pld_mode_type_attrs pld in
-    pp f "@[<2>%a%s%s:@;%a@;%a@]"
+    pp f "@[<2>%a%(%)%s:@;%a@;%a@]"
```

Comment:
I'd vote for having `pld_mode_type_attrs` return a `(Format.formatter -> unit) * _` rather than a format string. I didn't know about `%(%)` and I wouldn't be surprised if others didn't either -- and I find it's not put to much effect here.

Reply:
Fixed in 836a1eef026b4063bf9c283c8285b0854420b585